src/buttons/button.scss

```diff
@@ -312,14 +312,34 @@
 }
 
 // Icon-only buttons
-.btn-icon {
-  display: inline-block;
-  // stylelint-disable-next-line primer/spacing
-  padding: 7px;
-  line-height: $lh-condensed-ultra;
-  vertical-align: middle;
+// default mobile sizing
+.btn--icon {
+  display: grid;
+  width: 48px;
+  height: 48px;
```

Comment:
@langermank I'd refrain from changing the icon-only button like this for touch devices... I think it'd become way too hard to maintain these if they go from `32px` to `48px` depending on the device.

Instead, I'd make `32px` the default size for icon-only buttons, with a `48px` touch target using [`minTouchTarget`](https://github.com/primer/css/blob/8ca5bff3836ec2bc7a8c3b657a58dc3c57198806/src/support/mixins/misc.scss#L37) when `point: coarse`.

For icon-only buttons that are placed horizontally side by side (where the padding may be too small and one touch area might intersect with another), we can enforce a bigger spacing between elements (from default `8px` to touch-friendly `16px`) once we have a generic container for handling these horizontal action placements ([ActionBar](https://github.com/github/primer/issues/84)?)